tests/python/contrib/test_verilator/test_mobilenet.py

```diff
@@ -220,6 +221,8 @@ def tmobilenet(lanes):
     lanes : Int
         The number of vector lanes.
     """
+    if skip_test():
```

Comment:
One small note about this approach, when compared to the `skipIf` is that if you have no Verilator and you run these tests, once you reach the `return`, you'll see `PASSED` in the report, which might give the illusion that the test ran and passed.

The `skipIf` will be a bit more precise and report `SKIPPED`, so it is clear that the test was not run.